cwltool/sandboxjs.py

```diff
@@ -18,7 +17,7 @@ class JavascriptException(Exception):
 
 _logger = logging.getLogger("cwltool")
 
-JSON = Union[Dict[Text, Any], List[Any], Text, int, long, float, bool, None]
+JSON = Union[Dict[Text, Any], List[Any], Text, int, int, float, bool, None]
```

Comment:
Oh, didn't know we duplicated this with https://github.com/common-workflow-language/cwltool/pull/287/files#diff-cb6110fc9c5f825722db0f9c57cb1335R28